eZ/Publish/Core/REST/Server/Tests/Input/Parser/Criterion/LogicalAndTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * File containing a test class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\REST\Server\Tests\Input\Parser\Criterion;
+
+use eZ\Publish\API\Repository\Values\Content;
+use eZ\Publish\Core\REST\Common\Input\ParsingDispatcher;
+use eZ\Publish\Core\REST\Server\Input\Parser;
+use eZ\Publish\Core\REST\Server\Tests\Input\Parser\BaseTest;
+
+class LogicalAndTest extends BaseTest
+{
+    /**
+     * Test parsing.
+     */
+    public function testParseLogicalAnd()
+    {
+        // This is what xml parser is right now creating out of
+        // <AND>
+        //   <ContentTypeIdentifierCriterion>author</ContentTypeIdentifierCriterion>
+        //   <ContentTypeIdentifierCriterion>book</ContentTypeIdentifierCriterion>
+        //   <Field>
+        //     <name>title</name>
+        //     <operator>EQ</operator>
+        //     <value>Contributing to projects</value>
+        //   </Field>
+        // </AND>
```

Comment:
We rather avoid multiple inline comments, please move that to the PhpDoc of this method above.

Reply:
Done